modules/trustxBidAdapter.js

```diff
@@ -7,6 +7,7 @@ import { config } from '../src/config.js';
 const BIDDER_CODE = 'trustx';
 const ENDPOINT_URL = 'https://sofia.trustx.org/hb';
 const NEW_ENDPOINT_URL = 'https://grid.bidswitch.net/hbjson?sp=trustx';
+const ADDITIONAL_SYNC_URL = 'https://x.bidswitch.net/sync?ssp=themediagrid';
```

Comment:
should this be ssp=trustx?

Reply:
hi Patrick,
no, all good with ssp=themediagrid
we're in the process of migrating trustx ssp to mediagrid pipes, so this allows enforcing user matching directly grid in addition to trustx and thus helps smoothening the migration